extensions/wikia/CommunityPage/CommunityPageSpecialController.class.php

```diff
@@ -41,14 +41,12 @@ public function index() {
 			'topContributors' => $this->sendRequest(
 				'CommunityPageSpecialController',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "CommunityPageSpecialController" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)